src/java/org/apache/cassandra/tools/nodetool/Import.java

```diff
@@ -80,6 +80,16 @@ public class Import extends NodeToolCmd
             description = "Copy data from source directories instead of moving them")
     private boolean copyData = false;
 
+    @Option(title = "fail_on_missing_index",
+            name = {"-f", "--fail-on-missing-index"},
+            description = "Fail import when SSTables do not contain SAI components")
```

Comment:
```suggestion
            description = "Require existing index components for SSTables w/ attached indexes")
```
Trying something that works w/ Doug's flag name in case we change it.